google/cloud/aiplatform/experimental/vertex_model/utils/source_utils.py

```diff
@@ -57,6 +64,38 @@ def _make_source(cls_source: str, cls_name: str, instance_method: str):
         between the user-written code and the string returned by this method is that
         the user has the option to specify a method to call from __main__.
     """
-    src = "\n".join(["import torch", "import pandas as pd", cls_source])
-    src = src + "if __name__ == '__main__':\n" + f"\t{cls_name}().{instance_method}()"
+    src = "\n".join(
+        [
+            "import torch",
+            "import pandas as pd",
+            "from google.cloud.aiplatform import training_util",
+            "from google.cloud.aiplatform.experimental.vertex_model.serializers import *",
```

Comment:
I don't think this would work because `deserializer.__name__` referenced below would be `"_deserialize_dataframe"` but we're importing `pandas` and `dataloaders` here so would need to reference as `pandas._deserialize_dataframe`.

As a current work around I would change this to:
`google.cloud.aiplatform.experimental.vertex_model.serializers.pandas import *`

We can follow up and make this more structured but please add a comment for the motivation for this work around.